src/components/Kite/Kite.module.css

```diff
@@ -0,0 +1,88 @@
+.Kite {
+  z-index: var(--rvr-zindex-kite);
+  visibility: hidden;
+  box-sizing: border-box;
+  position: fixed;
+  top: 12px;
+  right: 12px;
+  transition: transform .6s ease-in-out;
```

Comment:
_No action needed for now_, but we should ask the design team about defining a set of standard transitions and box shadows.

My gut instinct is that Kites should probably use the same shadows as modals. They're all things that have similar perceived "altitudes" and visual weights, maybe?

Seems like our CSS variables could have something like a low-altitude / low contrast box shadow for tooltips and dropdown menus, and a high-altitude / high contrast box shadow for elements that float "further away" from the page content.

For transitions, we'd need to think a bit more. There's a heuristic for moving smaller items more slowly than larger items, and it would probably depend on whether the item is moving or just fading in/out. Regardless, it would be nice to have a formula and a few useful variables so we're consistent in our approach across components.